www/package.json

```diff
@@ -38,7 +38,8 @@
     "rehype-slug": "^4.0.1",
     "sass": "^1.53.0",
     "sass-loader": "12.6.0",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
```

Comment:
[curious] The `www/package-lock.json` file appears to be newly generated in this PR. Given the repo is now using NPM workspaces, I don't believe an individual package like `www` should have a `package-lock.json` in it, given its packages get listed/hoisted into the root `package-lock.json` file.

Should be able to delete the generated `www/package-lock.json` file and run `npm i` in the root of the repo?